media/css/pebbles/components/_footer.scss

```diff
@@ -7,6 +7,7 @@
 #colophon {
     @include open-sans;
     background: #000;
+    clear: both;
```

Comment:
Not for this PR.

I added it because the first time I opened my new template the entire page was black 😖 

The page behaves properly without it now but this seems like a good win for avoiding unforeseen issues on pages with coding problems.

![screen shot 2018-03-15 at 14 43 08-fullpage](https://user-images.githubusercontent.com/854701/37492754-bfa71aee-285f-11e8-9a43-113283928532.png)